addons/hr_holidays/views/hr_leave_allocation_views.xml

```diff
@@ -355,7 +355,7 @@
         <field name="name">All Allocations</field>
         <field name="res_model">hr.leave.allocation</field>
         <field name="view_mode">tree,kanban,form,activity</field>
-        <field name="context">{}</field>
+        <field name="context">{'force_allocation_type':1}</field>
```

Comment:
Using context key to implement a business behavior should be avoided. If we miss the context key, we miss the validation error.